src/testplane/testplane.ts

```diff
@@ -61,7 +61,13 @@ export class Testplane {
         }
     }
 
-    public async installDependencies() {
+    public async installDependencies({ primaryCacheHit = false } = {}) {
+        if (primaryCacheHit) {
+            core.debug("Skip installing dependencies as it was fully restored from cache");
+
+            return;
+        }
```

Comment:
`install-deps` makes network requests to some services, which could be unstable.
We don't need to call "install-deps" (and accordingly to make those requests) if we hit primary cache key